# This diff git patch modifies the package.json file to build a non-portable version of the application.
diff --git a/package.json b/package.json
index 40f3820..fe57b8c 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     "dev:lite": "check-node-version --node ^14.17 && bash ./apply-monkeypatches.sh && vue-cli-service electron:serve",
     "dev:stable": "check-node-version --node ^14.17 && bash ./apply-monkeypatches.sh && vue invoke electron-builder && npx dotenv-cli -e .env vue-cli-service electron:serve",
     "electron:build": "vue-cli-service electron:build",
-    "electron:build-linux-portable": "check-node-version --node ^14.17 && bash check-killme.sh && bash check-package.json.sh && bash ./apply-monkeypatches.sh && npx mkdirp RELEASE && node set-portable --portable=true && vue-cli-service electron:build --linux && mv dist_electron/linux-unpacked media-hoarder-portable && npx bestzip RELEASE/media-hoarder-VERSION-linux-x64-portable.zip media-hoarder-portable && npx rimraf media-hoarder-portable && node set-portable --portable=false && node set-release-version.js && rimraf dist_electron",
+    "electron:build-linux-non-portable": "check-node-version --node ^14.17 && bash check-killme.sh && bash check-package.json.sh && bash ./apply-monkeypatches.sh && npx mkdirp RELEASE && node set-portable --portable=false && vue-cli-service electron:build --linux && mv -v dist_electron/linux-unpacked RELEASE/media-hoarder_linux && node set-release-version.js",
     "electron:build-linux-setup": "check-node-version --node ^14.17 && bash check-killme.sh && bash check-package.json.sh && bash ./apply-monkeypatches.sh && npx mkdirp RELEASE && node set-portable --portable=false && vue-cli-service electron:build --linux deb appimage snap rpm && mv dist_electron/*.deb RELEASE/media-hoarder-VERSION-linux-x64.deb && mv dist_electron/*.AppImage RELEASE/media-hoarder-VERSION-linux-x64.AppImage && mv dist_electron/*.snap RELEASE/media-hoarder-VERSION-linux-x64.snap && mv dist_electron/*.rpm RELEASE/media-hoarder-VERSION-linux-x64.rpm && node set-release-version.js && rimraf dist_electron",
     "electron:build-mac": "check-node-version --node ^14.17 && bash check-killme.sh && bash check-package.json.sh && bash ./apply-monkeypatches.sh && npx mkdirp RELEASE && node set-portable --portable=false && vue-cli-service electron:build --mac dmg && mv dist_electron/*.dmg RELEASE/media-hoarder-VERSION-mac.dmg && node set-release-version.js && rimraf dist_electron",
     "electron:build-rpi": "check-node-version --node ^14.17 && bash check-killme.sh && bash check-package.json.sh && bash ./apply-monkeypatches.sh && vue-cli-service electron:build --dir --linux --armv7l deb appimage",
